src/Footer/index.js

```diff
@@ -1,10 +1,2 @@
-import React from 'react';
-import './Footer.css';
+export { default } from 'src/Footer/Footer';
```

Comment:
don't use absolute import here :) as it's part of your component module

Reply:
Updated + Pushed